target/xilinx/xilinx_ips/carfield_ip/tcl/run.tcl

```diff
@@ -20,13 +20,14 @@ source tcl/add_sources.tcl
 add_files -fileset constrs_1 constraints/ooc_carfield_ip.xdc
 set_property USED_IN {synthesis out_of_context} [get_files constraints/ooc_carfield_ip.xdc]
 add_files -fileset constrs_1 constraints/carfield_ip.xdc 
+add_files -fileset constrs_1 constraints/carfield_islands.tcl
 
 # Package IP
 set_property top carfield_xilinx_ip [current_fileset]
 
 # Attention SFCU is only used because of Carfield's structure
 update_compile_order -fileset sources_1
-synth_design -rtl -name rtl_1 -sfcu
+#synth_design -rtl -name rtl_1 -sfcu
```

Comment:
Quicker tests, maybe to be removed

Reply:
Should we remove this or not? I guess you need the synthesis step to package the IP right?